fuzzers/adacc-pure-concolic/fuzzer.py

```diff
@@ -0,0 +1,53 @@
+import os
+import shutil
+import subprocess
+
+from fuzzers import utils
+from fuzzers.afl import fuzzer as afl_fuzzer
+
+
+def build():
+    # Set flags to ensure compilation with SymCC
+    os.environ['CC'] = "/symcc/build/symcc"
+    os.environ['CXX'] = "/symcc/build/sym++"
+    os.environ['CFLAGS'] = ""
```

Comment:
any reason you're resetting the flags? 
Just wondering if this it going to make the resulting binary slower if optimization are disabled...

Reply:
Most likely, however, it will also improve in other cases as we don't need to have certain coverage sanitizer flags since SymCC works independently of those. In general, I think this is a good working PR, however, certain optimisations can be made. I would prefer to do those incrementally.